### PR TITLE
Add yosensi folder with all devices

### PR DESCRIPTION
**Pull request by yosensi**
#### Summary

Add folder with all devices with yamal files

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/yosensi/yo-360-profile-au.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-360-profile-au.yaml:15]: @yosensi if MAC version is 1.0.3 `Regional Parameters` might be: `RP001-1.0.3-RevA` check which one is the correct one and change it

File: vendor/yosensi/yo-360-profile-us.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-360-profile-us.yaml:15]: same case `MAC` and `Regional Parameters`

File: vendor/yosensi/yo-agribox-profile-au.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-agribox-profile-au.yaml:15]: review it

File: vendor/yosensi/yo-agribox-profile.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-agribox-profile.yaml:15]: review

File: vendor/yosensi/yo-airflow-pro-profile-us.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-airflow-pro-profile-us.yaml:15]: check

File: vendor/yosensi/yo-airflow-pro-profile.yaml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-airflow-pro-profile.yaml:15]: check

File: vendor/yosensi/yo-analog-profile-au.yaml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-analog-profile-au.yaml:15]: Review

File: vendor/yosensi/yo-analog-profile-us.yaml [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-analog-profile-us.yaml:15]: Review

File: vendor/yosensi/yo-analog-profile.yaml [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-analog-profile.yaml:15]: Review

File: vendor/yosensi/yo-distance-profile-au.yaml [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-distance-profile-au.yaml:15]: Review

File: vendor/yosensi/yo-distance-profile-us.yaml [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-distance-profile-us.yaml:15]: Review

File: vendor/yosensi/yo-h2o-profile-au.yaml [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-h2o-profile-au.yaml:15]: Review

File: vendor/yosensi/yo-h2o-profile-us.yaml [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.3'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-h2o-profile-us.yaml:15]: Review

File: vendor/yosensi/yo-h2o.yaml [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+    numeric: 1
+ 
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '3.1.0'
+    numeric: 1
+    hardwareVersions:
+      - '1.3'
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-h2o.yaml:15]: check the hardware version

File: vendor/yosensi/yo-pure-pro.yaml [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '3.3.0'
+    numeric: 1
+<<<<<<< HEAD
+=======
+    # Corresponding hardware versions (optional)
+>>>>>>> b13e0bc5a0e9016e8a419fc125a3f6c2f96cae58
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-pure-pro.yaml:17]: Review

**Comment by Jaime-Trinidad** [comment]
@yosensi please review the comments and check all your profile files, between the `MAC version` and `Regional Parameters`

**Comment by yosensi** [comment]
@Jaime-Trinidad I made review and changed MAC version for all devices. I added by accident .cpp file to one folder. I already did second commit that delete those files.

**Comment by yosensi** [comment]
Hello is there something more I need to do to upload Yosensi folder?

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**